pkg/resource/table/hooks_continuous_backup.go

```diff
@@ -0,0 +1,83 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//     http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package table
+
+import (
+	"context"
+
+	ackrtlog "github.com/aws-controllers-k8s/runtime/pkg/runtime/log"
+	svcsdk "github.com/aws/aws-sdk-go/service/dynamodb"
+
+	"github.com/aws-controllers-k8s/dynamodb-controller/apis/v1alpha1"
+)
+
+// syncContinuousBackup syncs the PointInTimeRecoverySpecification of the dynamodb table.
+func (rm *resourceManager) syncContinuousBackup(
+	ctx context.Context,
+	desired *resource,
+) (err error) {
+	rlog := ackrtlog.FromContext(ctx)
+	exit := rlog.Trace("rm.syncContinuousBackup")
+	defer func(err error) { exit(err) }(err)
+
+	pitrSpec := &svcsdk.PointInTimeRecoverySpecification{}
+	if desired.ko.Spec.ContinuousBackups != nil &&
+		desired.ko.Spec.ContinuousBackups.PointInTimeRecoveryEnabled != nil {
+		pitrSpec.SetPointInTimeRecoveryEnabled(
+			*desired.ko.Spec.ContinuousBackups.PointInTimeRecoveryEnabled,
+		)
+	}
+
```

Comment:
Not sure how this API works, but if a user deletes the field `ContinuousBackups` should we also explicitly set `SetPointInTimeRecoveryEnabled` to false? thinking about something like below, thoughts?
```go
    pitrSpec.SetPointInTimeRecoveryEnabled(*false)
    // only set the PITR is a user explicitly passes the full struct with enabled=true
	if desired.ko.Spec.ContinuousBackups != nil &&
		desired.ko.Spec.ContinuousBackups.PointInTimeRecoveryEnabled != nil {
		pitrSpec.SetPointInTimeRecoveryEnabled(
			*desired.ko.Spec.ContinuousBackups.PointInTimeRecoveryEnabled,
		)
	}
```

Reply:
the `pitrSpec.SetPointInTimeRecoveryEnabled(*false)` is already happening implicitly. If pitr is enabled and the user removes the entire `ContinuousBackups` field, then pitr is already disabled today